src/main/java/com/example/Storehouse/controllers/MainController.java

```diff
@@ -0,0 +1,43 @@
+package com.example.Storehouse.controllers;
+
+import com.example.Storehouse.models.request.Refund;
+import com.example.Storehouse.models.request.RequestMultiProducts;
+import com.example.Storehouse.service.StorehouseService;
+import io.swagger.annotations.Api;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+
+@RestController
+@Api(description = "Storehouse")
+public class MainController {
+    @Autowired
+    private StorehouseService service;
+
+
+    @GetMapping("/getProduct")
+    public ResponseEntity<?> getProduct(@RequestParam("idRequest") UUID id, @RequestParam("name") String name, @RequestParam("quantity") int quantity) {
```

Comment:
instead of Request params, use pls RequestBody and Proper request object

Reply:
done